types/ng-file-upload/index.d.ts

```diff
@@ -268,6 +268,10 @@ declare module 'angular' {
              * @type {string}
              */
             arrayKey?: string;
+            /**
+             * Add which HTTP method to use: 'POST' or 'PUT'(html5)
```

Comment:
Please add a space in between: `'PUT' (HTML5)`.

Reply:
Nice catch, fixed.

Reply:
I still see it the way it was before :(

Reply:
Fixed. My bad, sorry ;-)

Reply:
No worries. Thanks for fixing!